store/store.go

```diff
@@ -1143,10 +1144,10 @@ func (s *Store) Download(name string, downloadInfo *snap.DownloadInfo, pbar prog
 	return w.Name(), w.Sync()
 }
 
+var downloadBackoffs = []time.Duration{113, 191, 331, 557, 929, 0} // 3 pₙ₊₁ ≥ 5 pₙ; last entry should be 0
```

Comment:
the risk of putting the zero at the beginning is that it stops being obviously different from the series.
I'll add a why to the comment.

I too cringe at `113ns * 1ms` being `113ms` and not `1.13 × 10⁻¹⁰ s²`... maybe I'll make them `[]int` again and convert when multiplying.